src/common/hooks/useChartHeightForBreakpoint.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import { useBreakpoint } from 'common/hooks';
+
+export default function useChartHeightForBreakpoint(): number {
+  const chartHeightDesktop = 375;
+  const chartHeightMobile = 250;
+  const [chartHeight, setChartHeight] = useState<number>(chartHeightDesktop);
+  const isMobile = useBreakpoint(600);
+
+  useEffect(() => {
+    if (isMobile) {
+      setChartHeight(chartHeightMobile);
+    } else {
+      setChartHeight(chartHeightDesktop);
+    }
+  }, [isMobile]);
```

Comment:
Not sure we need the `useState()` and `useEffect()`...  I think we could just do:
```
  const isMobile = useBreakpoint(600);
  return isMobile ? chartHeightMobile : chartHeightDesktop;
```